Experiments/Experiments.xcodeproj/project.pbxproj

```diff
@@ -247,6 +250,7 @@
 			isa = PBXResourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				3FA7D9FC2D547E1700CE5611 /* Experiments.xcconfig in Resources */,
```

Comment:
The `.xcconfig` files should not appear as `PBXBuildFile` in the project nor be added to `PBXResourcesBuildPhase`: there's no point in including the `.xcconfig` file as a resource inside the final `.app` bundle, as it's only a config file used during the build.

Reply:
Of course. Thank you for noticing this, since I had obviously missed it.

I fixed it in 655c41c.

The thing that surprises me is that Xcode added them only to _three_ of the target/projects I touched. What's the logic behind that? I'm pretty sure I used the same approach to add the file to each: select the framework target, cmd n, add the file 🤔 🤷‍♂️